lib/cli.js

```diff
@@ -29,6 +29,7 @@ const EXIT_CODE_ERROR = 2;
  * @property {string} [color]
  * @property {string} [customFormatter]
  * @property {boolean} [disableDefaultIgnores]
+ * @property {string} [ext=".css"]
```

Comment:
[suggest] We may be able to omit the default value here to avoid repetition.

```suggestion
 * @property {string} [ext]
```